scripts/delayed.js

```diff
@@ -33,3 +33,35 @@ if (window.location.host.includes('partsasist.com')) {
 fireGTM(window, document, 'script', 'dataLayer', 'GTM-NRP6TBV');
 window.OptanonWrapper = () => {};
 // OneTrust Cookies Consent Notice end
+
+// Contact form campaign Start
+
+// Check for Campaign & Business Unit IDs in metadata & Pardot form block on the page
+// then add the IDs to the script that will be injected.
+const campaignMeta = document.querySelector('meta[name="campaign-id"]');
+const businessUnitMeta = document.querySelector('meta[name="business-unit-id"]');
+const pardotForm = document.querySelector('.pardot-form.block');
+
+if (campaignMeta && businessUnitMeta && pardotForm) {
+  const campaignId = campaignMeta.getAttribute('content');
+  const businessUnitId = businessUnitMeta.getAttribute('content');
+  // create a fragment to hold the script
+  const pardotScript = document.createRange().createContextualFragment(`
+    <script type="text/javascript">
+      piAId = '${businessUnitId}';
+      piCId = '${campaignId}';
+      piHostname = '[pi.pardot.com](http://pi.pardot.com/)';
```

Comment:
Not sure if that `[pi.pardot.com](http://pi.pardot.com/)` it might be a markdown notation.